<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenCVPanel</class>
 <widget class="QWidget" name="OpenCVPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>123</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="11" column="1">
    <widget class="QLabel" name="mLabel_KSize">
     <property name="minimumSize">
      <size>
       <width>15</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLabel" name="gLabel_KSize">
     <property name="minimumSize">
      <size>
       <width>15</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QSlider" name="mSlider_KSize">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>9</number>
     </property>
     <property name="singleStep">
      <number>2</number>
     </property>
     <property name="pageStep">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="18" column="0" colspan="2">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>aperatureSize</string>
     </property>
    </widget>
   </item>
   <item row="26" column="0" colspan="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="21" column="1">
    <widget class="QCheckBox" name="hedCheckBox_OnOff">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <widget class="QSlider" name="cSlider_Threshold1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximum">
      <number>400</number>
     </property>
     <property name="singleStep">
      <number>2</number>
     </property>
     <property name="pageStep">
      <number>4</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="17" column="0">
    <widget class="QSlider" name="cSlider_Threshold2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximum">
      <number>400</number>
     </property>
     <property name="singleStep">
      <number>2</number>
     </property>
     <property name="pageStep">
      <number>4</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QSlider" name="gSlider_KSize">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>31</number>
     </property>
     <property name="singleStep">
      <number>2</number>
     </property>
     <property name="pageStep">
      <number>2</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
     <property name="sliderPosition">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickInterval">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="19" column="1">
    <widget class="QLabel" name="cLabel_AperatureSize">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="QLabel" name="label_06">
     <property name="text">
      <string>KSize</string>
     </property>
    </widget>
   </item>
   <item row="20" column="0">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="label_07">
     <property name="text">
      <string>Canny Filter</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label_01">
     <property name="text">
      <string>Resize</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QSlider" name="sizeDividor_Slider">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>8</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="sizeDividor_Label">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="14" column="0" colspan="2">
    <widget class="QLabel" name="label_08">
     <property name="text">
      <string>threshold1</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="label_04">
     <property name="text">
      <string>KSize</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="QCheckBox" name="cCheckBox_OnOff">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="Line" name="line_1">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_02">
     <property name="text">
      <string>Size dividor</string>
     </property>
    </widget>
   </item>
   <item row="23" column="1">
    <widget class="QLabel" name="hedLabel_BlockSize">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="16" column="0" colspan="2">
    <widget class="QLabel" name="label_09">
     <property name="text">
      <string>threshold2</string>
     </property>
    </widget>
   </item>
   <item row="25" column="1">
    <widget class="QLabel" name="hedLabel_C">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="24" column="0" colspan="2">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>C</string>
     </property>
    </widget>
   </item>
   <item row="19" column="0">
    <widget class="QSlider" name="cSlider_AperatureSize">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimum">
      <number>3</number>
     </property>
     <property name="maximum">
      <number>7</number>
     </property>
     <property name="singleStep">
      <number>2</number>
     </property>
     <property name="pageStep">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="25" column="0">
    <widget class="QSlider" name="hedSlider_C">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximum">
      <number>5</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="23" column="0">
    <widget class="QSlider" name="hedSlider_BlockSize">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimum">
      <number>3</number>
     </property>
     <property name="maximum">
      <number>101</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QLabel" name="label_05">
     <property name="text">
      <string>Median Blur</string>
     </property>
    </widget>
   </item>
   <item row="22" column="0" colspan="2">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>block size</string>
     </property>
    </widget>
   </item>
   <item row="15" column="1">
    <widget class="QLabel" name="cLabel_Threshold1">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="12" column="0" colspan="2">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="21" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>HED</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="label_03">
     <property name="text">
      <string>Gaussian Blur</string>
     </property>
    </widget>
   </item>
   <item row="17" column="1">
    <widget class="QLabel" name="cLabel_Threshold2">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
